### DISCUSSION TITLE
How To Fix this Issue ?

### DISCUSSION THREAD

**Discussion by syedmahroof** [danjohnson95/laravel-pinout#1] [General]
composer require danjohnson95/pinout

In PackageDiscoveryTrait.php line 339:
                                                                               
  Could not find a version of package danjohnson95/pinout matching your minim  
  um-stability (stable). Require it with an explicit version constraint allow  
  ing its desired stability.
  
  Using Fresh Laravel Installation
  
<img width="564" alt="Screenshot 2024-09-03 at 6 17 17 PM" src="https://github.com/user-attachments/assets/99b612cc-2331-426c-a1ba-91c514852473">

**Comment by danjohnson95** [comment]
Hi there! Thanks for pointing this out.

I haven't tagged a stable version of this project yet, so in the meantime you can target the `main` branch.

```
composer require danjohnson95/pinout:dev-main
```

**Reply by syedmahroof** [reply]
Hi there!

Thanks for your quick response. I followed your suggestion and installed the package,
Everything is working perfectly now. I appreciate your help!

Thanks again!

**Reply by danjohnson95** [reply]
Amazing! Have fun, and please do share anything you build!
